### PR TITLE
added reward event

### PR DESCRIPTION
**Pull request by Kazuma-47**
Added features:

-Added Unity event that will be invoked when the rewared ad is finished

### CODE REVIEW

File: Assets/Scripts/RewardedAdHandler.cs [hunk 1]
```diff
@@ -14,6 +15,8 @@ public class RewardedAdHandler : MonoBehaviour, IUnityAdsLoadListener, IUnityAds
     [SerializeField] string _androidAdUnitId = "Rewarded_Android";
     [SerializeField] string _iOSAdUnitId = "Rewarded_iOS";
 
+    [Header("Implementation")]
+    [SerializeField] private UnityEvent OnReward = new UnityEvent();
     private bool debugMode;
```
Comment (DanielNijkamp) [inline Assets/Scripts/RewardedAdHandler.cs]:
```suggestion
    [SerializeField] private UnityEvent _onReward = new UnityEvent();
    private bool _debugMode;
```

**Review by DanielNijkamp — CHANGES_REQUESTED** [review]
Check nog snel private variables voor de rest is gwn prim

**Comment by DanielNijkamp** [comment]
Ook nog miss alle script gereleteerd tot ads in hun eigen folder zetten binnen `Scripts`. dus `Scripts` -> `Ads`

File: Assets/Scripts/RewardedAdHandler.cs (hunk 1 above)
Comment (Kazuma-47) [reply Assets/Scripts/RewardedAdHandler.cs]: refactored and checked the private variables in code

**Comment by Kazuma-47** [comment]
> Ook nog miss alle script gereleteerd tot ads in hun eigen folder zetten binnen `Scripts`. dus `Scripts` -> `Ads`

Done :))

**Review by DanielNijkamp — APPROVED** [review]

**Pull request merged**